release-content/0.15/release-notes/15204_box_shadow.md

```diff
@@ -1,4 +1,37 @@
 <!-- box shadow -->
 <!-- https://github.com/bevyengine/bevy/pull/15204 -->
 
-<!-- TODO -->
+![A demonstration of Bevy's box shadows. There are 12 white shapes on a salmon pink background, and the border radius, aspect ratio and softness of the shadows is varied for each of them. The shadows cause the buttons to appear to hover above the page.](box_shadow.png)
+
+Bevy's UI now supports box shadows!
+When designing user interfaces, creating a sense of depth can provide cues to users about which elements are interactable.
+One of the key features of elements with depth (that are not just flat on the metaphorical page) is that they cast shadows.
+
+UI shadows aren't intended to be realistic: no raytracing here!
+Instead, we care about offering a pretty effect with maximum artistic control at a minimum runtime cost.
+
+By adding the new [`BoxShadow`] component to any [`Node`] entity, you can draw a pretty shadow behind your widgets.
+
+```rust
+#[derive(Component)]
+pub struct BoxShadow {
+    /// The shadow's color
+    pub color: Color,
+    /// Horizontal offset
+    pub x_offset: Val,
+    /// Vertical offset
+    pub y_offset: Val,
+    /// Horizontal difference in size from the occluding uninode
```

Comment:
Just noticed this is wrong. The original draft of the PR had separate horizontal and vertical spread values but that would have needed some sort of elliptical border radius support so it was removed but I must have missed this comment in the PR description. From the doc comments for the struct:

```suggestion
    /// How much the shadow should spread outward.
    ///
    /// Negative values will make the shadow shrink inwards.
    /// Percentage values are based on the width of the UI node.
```